virtool/subtractions/db.py

```diff
@@ -12,10 +16,69 @@
     "job",
     "name",
     "nickname",
-    "user"
+    "user",
+    "has_file"
 ]
 
 
+class CreateSubtractionTask(virtool.tasks.task.Task):
```

Comment:
Change to `WriteSubtractionFASTATask` and `write_subtraction_fasta`.

Currently, this can cause confusion with the job called `CreateSubtraction`